src/xrc-tests/src/mock_responses/forex/nepal.rs

```diff
@@ -41,44 +43,44 @@ pub fn build_response_body(timestamp: u64) -> ResponseBody {
                                 "name": "U.S. Dollar",
                                 "unit": 1
                             },
-                            "buy": "125.05",
-                            "sell": "125.65"
+                            "buy": rates.get("USD_BUY").cloned().unwrap_or("125.05"),
+                            "sell": rates.get("USD_SELL").cloned().unwrap_or("125.65")
                         },
                         {
                             "currency": {
                                 "iso3": "EUR",
                                 "name": "European Euro",
                                 "unit": 1
                             },
-                            "buy": "132.37",
-                            "sell": "133.00"
+                            "buy": rates.get("EUR_BUY").cloned().unwrap_or("132.37"),
+                            "sell": rates.get("EUR_BUY").cloned().unwrap_or("133.00")
```

Comment:
Shouldn't it be just "EUR" or - if these keys exist - "EUR_BUY" and "EUR_SELL"?
The same question applies to the currencies below.